app/components/gh-members-chart.js

```diff
@@ -18,7 +18,9 @@ export default Component.extend({
     stats: null,
     chartData: null,
     chartOptions: null,
-    startDateLabel: '',
+    startDateLabel: computed('membersStats.days', function () {
+        return moment(new Date()).add(-this.membersStats.days + 1, 'days').format(DATE_FORMAT);
```

Comment:
Unfortunately this won't work when the "All time" range is chosen because `memberStats.days` will be `"all-time"` and not a number.

What you could do instead is to use `memberStats.stats` as the dependent key and use `Object.keys(memberStats.stats.total_on_date)[0]` to grab the first date in the range and build the label from there.

Reply:
ahh, thank you for that!